src/app/components/TransactionsTable/index.tsx

```diff
@@ -27,7 +27,13 @@ export default function TransactionsTable({
       <Loading />
     </div>
   ) : !transactions?.length && noResultMsg ? (
-    <p className="text-gray-500 dark:text-neutral-400">{noResultMsg}</p>
+    <p
+      className="text-gray-500 dark:text-neutral-400"
+      tabIndex={0}
+      aria-label={noResultMsg}
```

Comment:
why do we add a label when the value is already visible and it's exactly the same? Is there any benefit?

Reply:
There are quite a few similar issues here. Could you re-review the changes and make sure the labels provided are actually helpful and necessary? then I think @stackingsaunter can hopefully test it.